aptos-move/framework/aptos-framework/sources/atomic_bridge.move

```diff
@@ -822,6 +822,20 @@ module aptos_framework::atomic_bridge_store {
         keccak256(combined_bytes)
     }
 
+    #[view]
+    public fun get_bridge_transfer_details(
+        bridge_transfer_id: vector<u8>
+    ): BridgeTransferDetails<address, EthereumAddress> acquires SmartTableWrapper {
+        let table = borrow_global<SmartTableWrapper<vector<u8>, BridgeTransferDetails<address, EthereumAddress>>>(@aptos_framework);
+
+        let details_ref = smart_table::borrow(
+            &table.inner,
+            bridge_transfer_id
+        );
+
+        *details_ref
+    }
+
```

Comment:
This would only provide details for transfers from L2 to L1 `<address, EthereumAddress>` or `<Initiator, Recipient>` I believe, https://github.com/movementlabsxyz/aptos-core/blob/70be3926ff79ff4cdb0cee928f717fafcd41ecdd/aptos-move/framework/aptos-framework/sources/atomic_bridge.move#L569

Maybe there is also no need to require the whole struct returned and a tuple could suffice